src/Monolog/Handler/RollbarHandler.php

```diff
@@ -54,7 +54,15 @@ public function __construct(RollbarNotifier $rollbarNotifier, $level = Logger::E
     protected function write(array $record)
     {
         if (isset($record['context']['exception']) && $record['context']['exception'] instanceof Exception) {
-            $this->rollbarNotifier->report_exception($record['context']['exception']);
+            $context = $record['context'];
+            unset($context['exception']);
+
+            $payload = isset($context['payload']) ? $context['payload'] : [];
+            if ($context['payload']) {
```

Comment:
Should this be `if (isset($context['payload']))`?

When the `isset` check above fails, `if ($context['payload'])` will cause an `undefined index` error, correct?